src/org/labkey/snd/query/AttributeDataTable.java

```diff
@@ -411,7 +411,7 @@ public int truncateRows(User user, Container container, @Nullable Map<Enum, Obje
 
             //Deleting event narrative cache
             log.info("Clearing event narrative cache.");
-            _sndService.clearNarrativeCache(container, user);
```

Comment:
Can we please add the same logging message ("Narrative cache will not be updated or Cleared) as in the SNDManager file.

Reply:
I agree. The log message on line 413 needs to be changed.

Reply:
Just going to remove the logging since nothing is happening here now